### DISCUSSION TITLE
Shell.TitleView not appearing at the correct position on iOS ?

### DISCUSSION THREAD

**Discussion by softlion** [dotnet/maui#13454] [General]
Anyone seeing that behavior ?
Why is the title view not centered vertically on iOS ? This works correctly on Android. 

```
<ContentPage ... >
<Shell.TitleView>
        <Grid BackgroundColor="Yellow">
            <Label Text="Hello Hello" FontSize="24" TextColor="Black" BackgroundColor="Red"  />
        </Grid>
    </Shell.TitleView>
```

![image](https://user-images.githubusercontent.com/190756/220152976-619d3c6e-038f-4bae-87f8-84e56ffbb2af.png)

**Comment by MichaelShapiro** [comment]
@softlion Add an explicit height to the Grid in your example: 
        <Grid  BackgroundColor="Yellow" HeightRequest="45"

Value of 45 should work good in portrait mode on iOS.

**Reply by softlion** [reply]
Instead I used the "PureWeen.Maui.FixesAndWorkarounds" nuget and that fixed it.
